src/nifax/control/util/Table.java

```diff
@@ -0,0 +1,32 @@
+package nifax.control.util;
+
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableModel;
+
+/**
+ *
+ * @author NB
+ */
+public final class Table {
+
+    public static void ocultColumns(JTable tbl, int columna[]) {
```

Comment:
word Ocult i think it doesn't not exist. It will be more clarily if you use hide, another different conjugation of the verb.